### DISCUSSION TITLE
so-allow command help

### DISCUSSION THREAD

**Discussion by larstchet** [Security-Onion-Solutions/securityonion#7934] [Unsupported Versions] [answered]
I've tried to find this information by searching the Internet but I'm not having any luck. I need to know when I run "so-allow" what file(s) are getting modified so I can see the changes being made. Yes, I'm fairly new to Linux and SO...Thanks, Larry

**Comment by lrhenness** [comment, marked as answer]
Hi there,

Take a look in `/opt/so/saltstack/default/salt/firewall/`

This directory should contain a couple of yaml files for the firewall configuration. 

Warning from the docs: 

> The files in this directory should not be modified as they could possibly be overwritten during a [soup](https://docs.securityonion.net/en/2.3/soup.html#soup) update in the event we update those files.

The relevant areas of the documentation are as follows:

- https://docs.securityonion.net/en/2.3/so-allow.html
- https://docs.securityonion.net/en/2.3/firewall.html#host-firewall

The second link there might of particular interest to you and your goals.
